psutil/net.go

```diff
@@ -75,13 +86,21 @@ func netIOCounters(nss []core.Namespace) ([]plugin.MetricType, error) {
 		return nil, err
 	}
 
+	// gather conntrack metric
+	conntrack, err := net.FilterCounters()
+	if err != nil {
+		// Conntrack probably not loaded into the kernel.
+		fmt.Printf("Requested netfilter statistics %s not found", err)
```

Comment:
Yes, with this plugin, metric list is "static", snap metric list return conntrack_count while it not available on the server. I think it good, to see this error in snap log and continue to retreive other metric of the task. With return statement, task will failed and no metric will be publish. This is why i remove the return statement.